src/domain/event/eventInfo/EventInfo.tsx

```diff
@@ -147,6 +184,18 @@ const EventInfo: React.FC<Props> = ({ event }) => {
           </Link>
         </InfoWithIcon>
 
+        {/* Audience */}
```

Comment:
There is new components showing in EventInfo.tsx (otherCourses, audience items, courseInfo rows) but these are not covered in EventInfo.test.tsx?
When the coverage is covered to pr, we could easily see that coverage has dropped for this component.

Reply:
more testing coverage added, other course or other event are not tested here as integration tests, but those components have tests  in itself